tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1692,152 +1660,162 @@ def test_mpu_during_pod_restart(self):
-                                                    log_prefix=self.test_prefix,
-                                                    skipcleanup=True)
+        LOGGER.info("Step 9.2: Download the uploaded objects in step 4 and verify checksum")
+        resp = self.ha_obj.dnld_obj_verify_chcksm(s3_test_obj, bucket_name_1, object_name_1,
+                                                  download_path1, upload_checksum2)
         assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 11: Successfully created s3 account and multiple buckets and ran IOs")
+        LOGGER.info("Step 9: Successfully downloaded the objects and verified the checksum")
 
+        object_name_2 = f"ha-mp-obj-{int(perf_counter_ns())}"
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("Create new bucket")
+            bucket_name_2 = f"ha-mp-bkt-{int(perf_counter_ns())}"
+        else:
+            bucket_name_2 = self.bucket_name
+        LOGGER.info("Step 10: Again perform multipart upload for size %s MB in total %s parts and "
+                    "download the object and verify checksum", file_size, total_parts)
+        resp = self.ha_obj.create_bucket_to_complete_mpu(s3_data=self.s3_clean,
+                                                         bucket_name=bucket_name_2,
+                                                         object_name=object_name_2,
+                                                         file_size=file_size,
+                                                         total_parts=total_parts,
+                                                         multipart_obj_path=self.multipart_obj_path)
+        assert_utils.assert_true(resp[0], resp)
+        result = s3_test_obj.object_info(bucket_name_2, object_name_2)
+        obj_size = result[1]["ContentLength"]
+        LOGGER.debug("Uploaded object info for %s is %s", bucket_name_2, result)
+        assert_utils.assert_equal(obj_size, file_size * const.Sizes.MB)
+        upload_checksum = str(resp[2])
+        LOGGER.info("Successfully performed multipart upload for  size %s MB in total %s parts.",
+                    file_size, total_parts)
+        resp = self.ha_obj.dnld_obj_verify_chcksm(s3_test_obj, bucket_name_2, object_name_2,
+                                                  download_path2, upload_checksum)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 10: Successfully performed multipart upload and downloaded the object "
+                    "and verified the checksum")
+        self.extra_files.extend((self.multipart_obj_path, download_path, download_path1,
+                                 download_path2))
         LOGGER.info("ENDED: Test to verify multipart upload during data pod restart")
 
+    # pylint: disable=too-many-branches
     @pytest.mark.ha
     @pytest.mark.lc
-    @pytest.mark.skip(reason="Blocked until F-22A is available")
```

Comment:
why this is removed. Copy object won't be suppoerted. Do we need to update this marked instead of removing

Reply:
copy object will be supported in PI-8 so we can remove the marker @ankita-93